kinds/structured/user/user_kind.cue

```diff
@@ -0,0 +1,40 @@
+package kind
+
+name: "User"
+maturity: "merged"
+
+lineage: seqs: [
+	{
+		schemas: [
+			// v0.0
+			{
+				// Email is the user's email.
+				email: string
+				// Name is the user's name.
+				name: string
+				// Login is the name used for login.
+				login: string
+				// Theme is Grafana theme used by the user.
+				theme?: string @grafanamaturity(MaybeRemove)
+				// OrgId is the org where the user belongs to.
+				orgId?: string @grafanamaturity(ToMetadata="sys")
+				// IsGrafanaAdmin indicates if the user belongs to Grafana.
+				isGrafanaAdmin: bool
+				// IsDisabled indicates if the user is disabled.
+				isDisabled: bool
+				// IsDisabled indicates if the user is external.
+				isExternal: bool
+				// AuthLabels is a list of authentication providers used (OAuth, SAML, LDAP...)
+				authLabels: [...string]
+				// CreatedAt indicates when the user was created.
+				createdAt: int64 @grafanamaturity(ToMetadata="sys")
+				// UpdatedAt indicates when the user was updated.
+				updatedAt: int64 @grafanamaturity(ToMetadata="sys")
+				// AvatarUrl is the user's avatar URL.
+				avatarUrl: string
+				// AccessControl metadata associated with a given resource.
+				accessControl: [string]: bool @grafanamaturity(ToMetadata="sys")
```

Comment:
For any reason, I cannot set this value as optional. The DTO [here](https://github.com/grafana/grafana/blob/main/pkg/services/user/model.go#L156) has the `omitempty` tag and adding the `?` should work. 

Is it a bug or is it intentional? @sdboyer

Reply:
`omitempty` has some weirdness to it - it doesn't actually correspond to optionality. The only way Go can really express field optionality is through a (nil) pointer. Some more discussion about that [here](https://github.com/grafana/grafana/pull/59621#discussion_r1037424816), and grafana/grok#1.

But when you say "adding the `?` should work, do you mean like this?

```cue
accessControl?: [string]: bool
```

Reply:
Yes, I mean that.

Reply:
OK, _that_ may be hitting yet a different known-but-not-yet-clearly-documented issue - i believe that we've encountered an issue where combining an optional field with a map has unexpected results.